### PR TITLE
chore(lighthouse): Add plugins to accessibility testing

### PR DESCRIPTION
**Pull request by adamdmharvey**
## Hey, I just made a Pull Request!

Using the excellent guidelines from https://backstage.io/docs/accessibility/, this PR adds a few additional plugins to the accessibility testing. All look pretty good so far, but it helps point out a few enhancements.

Also minor updates to the instructions.

| Example 1 | Example 2 | Example 3 |
| --- | --- | --- |
|  image | image | image |

PS: `@lhci/cli` now has a v0.12 released which bumps to Lighthouse v10. However, when trying it it seemed to think I didn't have Chrome installed, though v0.11 worked just fine. So I avoided doing that bump for now.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by emmaindal — APPROVED** [review]
amazing, thanks a lot for this contribution! 💯 

Optional: Would you be open to help us to open GitHub issues for the accessibility issues that made them not reach 100? That way we can add a `help-wanted` label to them and hopefully get those issues fixed as well 🙏

**Comment by adamdmharvey** [comment]
@emmaindal I can definitely do that!

A sidebar query: while I was there, I turned on the `best-practices` feature as well and it had some good tips. Has there been any discussion on that?

```
assert: {
      assertions: {
        'categories:performance': 'off',
        'categories:pwa': 'off',
        'categories:best-practices': ['warn', { minScore: 0.75 }],
        'categories:seo': 'off',
        'categories:accessibility': ['error', { minScore: 0.95 }],
      },
    },
```

**Comment by emmaindal** [comment]
Thank you so much for opening issues @adamdmharvey, very much appreciated! could call out with the best practises, personally I wouldn't be against enabling best practises as well, but will leave that up to @freben and team as we in @backstage/discoverability-maintainers focus (amongst other things) on trying to improve the accessibility of the overall Backstage experience, but will probably not have time to focus on making changes aligned with the suggestions from best practises as of now.

File: docs/accessibility/index.md
Comment (freben) [inline docs/accessibility/index.md:27]: you seem to have added this twice

**Review by freben — COMMENTED** [review]
Yeah I don't mind adding tests and issues that help guide the community toward further improving accessibility overall, but want to still highlight that the maintainers themselves don't work on this at this point.

File: docs/accessibility/index.md
Comment (adamdmharvey) [reply docs/accessibility/index.md:27]: thx for the review @freben ; can you elaborate?

here in this docs page is the *example config* for others to reference. what i did was move the plugins suggestion down below the main parts, because likely what we'd want is users to add their plugins at the BOTTOM of the list, rather than at the top above all other settings in the example?

this whole section needed a bit of indentation to format a bit better, so a lot of the lines look modified but it's just whitespace.

If you look at it with whitespace off it is quite a bit easier:

image

(remove from up top, add below, but i'm ADDING a diff Markdown section so there are extra plus signs :) )

**Review by freben — APPROVED** [review]

**Pull request merged**
